cli-commands/commands.js

```diff
@@ -61,6 +62,20 @@ const commands = [
 		commandProcessor: (argv) => {
 			history.run(argv.limit, console);
 		}
+	},
+	{
+		command: 'compile [default_path]',
+		description: 'Compiles smart contracts',
```

Comment:
I think this should be better described. Like:
Compiles the smart contracts that are in the directory `contracts` in the path provided by the `dir` parameter (defaults to `.`).